pro_tes/middleware/task_distribution/distance.py

```diff
@@ -71,7 +69,21 @@ def get_uri_combination(
     Args:
         input_uri: List of input uris of TES request.
         tes_uri: List of TES instance.
+
     Returns:
+        An AccessUriCombination object, consisting of the following keys:
```

Comment:
Refer to `AccessUriCombination` via a fully qualified path, e.g., `pro_tes.models.AccessUriCombination` (I didn't check this, there may be other submodules). Then there is no need to described that object, people can just look it up.

Even better: you can use Sphinx shortcuts to refer to other functions, methods, modules etc: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

This is really powerful!

In this case, it would be something like:

```markdown
:class:`pro_tes.models.AccessUriCombination` 
```

(you can omit the `:py` at the start, see https://stackoverflow.com/a/22714510)

including backticks and all.

Reply:
Done